ci/release_notes.md

```diff
@@ -0,0 +1,25 @@
+# Improvements
+
+- Genesis now supports the Genesis Index, a free service available
+  online at https://genesis.starkandwayne.com that tracks releases
+  and stemcells from bosh.io and Github.  Genesis now uses this to
+  supply version, tarball URL and SHA1 checksum information for
+  both releases and stemcells.
+
+  This makes it possible for newer BOSH CLIs to transparently
+  upload missing releases and stemcells to your BOSH director,
+  which in turn enables Concourse pipelines to easily roll
+  stemcell and release upgrades through unattended.
+
+  One side-effect of this change is that the "latest" version
+  keyword now no longer means "latest available on the BOSH
+  director", but instead "latest known to the index".
+
```

Comment:
This concerns me, since 'latest' now means two different things to your manifest, depending on what toolset users are using.  I don't like changing conventions subtly on people, and I'm not sure how well this will go over in an environment under strict change control